aws-cpp-sdk-core/source/utils/crypto/commoncrypto/CryptoImpl.cpp

```diff
@@ -22,6 +22,11 @@
 #include <CommonCrypto/CommonCryptor.h>
 #include <CommonCrypto/CommonSymmetricKeywrap.h>
 
+// Fix for deprectation errors in OSX 10.15: https://github.com/aws/aws-sdk-cpp/issues/1259
+#if defined(__APPLE__) && defined(__clang__)
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
+#endif
```

Comment:
Would you mind adding `push` and `pop`? When doing uber builds, we will put all source files together, this way, the warnings will be ignored all the way to other source files.

Reply:
Added now in 76ce92f